packages/villain/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=villain
+pkgver=324.58a8c02
+pkgrel=1
+pkgdesc='C2 framework that can handle multiple TCP socket & HoaxShell-based reverse shells, enhance their functionality with additional features and share them among connected sibling servers.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-networking')
+url='https://github.com/t3l3machus/Villain'
+license=('CCBY-NC-ND-4.0')
+depends=('python' 'python-netifaces' 'python-pycryptodome' 'python-pyperclip' 'python-requests' 'python-gnureadline')
+makedepends=('git')
+source=("$pkgname::git+https://github.com/t3l3machus/Villain.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
Could you rather use that: https://github.com/BlackArch/blackarch-pkgbuilds/blob/a6f1e0daf6683ff186ac4fbd34fcd024a082c2f4/PKGBUILD-generic-git#L17-L23?
tags are availables

Reply:
Fixed